core/controllers/reader.py

```diff
@@ -291,7 +348,12 @@ class StateHitEventHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """Handles POST requests.
+        Records learner's new state
+        Args:
+            exploration_id: str. The id of the exploration the learner wants
```

Comment:
`The ID of the exploration.` should suffice (given the docstring suggestion above).